lib/logger.js

```diff
@@ -131,7 +131,8 @@ const logger = function (level, files, noStdout, prefix) {
 
     files = files || [];
 
-    const isNpm = (__dirname.replace(/\\/g, '/').toLowerCase().indexOf('node_modules/' + tools.appName.toLowerCase() + '.js-controller') !== -1);
+    // indicator which is used to determine the log dir for developing, where it should be inside the repository
+    const isNpm = (__dirname.replace(/\\/g, '/').toLowerCase().indexOf(`node_modules/${tools.appName.toLowerCase()}.js-controller`) !== -1);
```

Comment:
we should moe away here too from "string.indexOf !== -1" to String.includes ... more easy readable

Reply:
Ok for me